### ISSUE TITLE
Update user.app_metadata

### ISSUE THREAD

**Issue by GShotwell** [auth0/react-native-lock#29]
I searched through the auth0 documentation for this and couldn't find it. What's the preferred way to update the user object from a RN application, in particular I'm interested in updating `user.app_metadata` the closest I found was how to accomplish this using auth0 rules, but I didn't see anything similar for the RN API.

Thanks for the great library,

**Comment by hzalaz** [comment]
@GShotwell it's just a request to this API https://auth0.com/docs/api/v2#!/Users/patch_users_by_id, we are working to make the node api client lib we have RN compatible but in the meantime you can try it with fetch API of ES6.

**Comment by GShotwell** [comment]
Thanks! The update is working okay, should we get the app metatdata in the same way, or is there a better solution with react-native lock, intuitively I would've thought this would be something like: 

`
    lock.show({closable:true}, (err, profile, token) => {
      if(err){
        return false;
      }
      console.log(profile.appMetadata.<metadata_object>)

};
`

But that doesn't see to work. 

Thanks again,

**Comment by changchup** [comment]
Is it also possible to get user.app_metadata information from the lock.show method
```javascript
  lock.show({
      closable:true,
      authParams: {
      scope: 'openid profile user_metadata app_metadata'
    }}, (err, profile, token) => {
```

but this only returns an empty user_metadata and app_metadata objects.

**Comment by hzalaz** [comment]
Should be fixed by #56 on iOS
